cdm/core/src/main/java/ucar/nc2/filter/Classifier.java

```diff
@@ -2,20 +2,26 @@
 
 import java.io.IOException;
 import ucar.ma2.Array;
-import ucar.ma2.DataType;
 import ucar.ma2.IndexIterator;
+import ucar.nc2.NetcdfFile;
+import ucar.nc2.Variable;
+import ucar.nc2.dataset.NetcdfDatasets;
 import ucar.nc2.dataset.VariableDS;
+import ucar.nc2.Attribute;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
 
 public class Classifier implements Enhancement {
   private Classifier classifier = null;
   private static Classifier emptyClassifier;
   private int classifiedVal;
-  private int[] classifiedArray;
+  private List<Attribute> AttCat;
+  private List<int[]> rules = new ArrayList<>();
 
   public static Classifier createFromVariable(VariableDS var) {
     try {
```

Comment:
this needs to be implemented to read the classifier attribute (`CDM.RANGE_CAT`) and use that to call `Classifier(List<Attribute>)`